### PR TITLE
Clarification about word locally

### PR DESCRIPTION
**Pull request by pherjung**
Instead of using local, be more explicit.

### CODE REVIEW

File: Reference/Sailfish_OS_Cheat_Sheet/README.md [hunk 1]
```diff
@@ -541,7 +541,7 @@ Import contacts from vCard
 devel-su -p vcardconverter contacts.vcf
 ```
 
-Export local contacts to vCard
+Export contacts stored in the phone memory to vCard
```
Comment (rainemak) [inline Reference/Sailfish_OS_Cheat_Sheet/README.md:544]:
```suggestion
Export contacts that are stored to the device internal memory to vCard
```
Not necessarily phone either.

File: Reference/Sailfish_OS_Cheat_Sheet/README.md [hunk 2]
```diff
@@ -541,7 +541,7 @@ Import contacts from vCard
 devel-su -p vcardconverter contacts.vcf
 ```
 
-Export local contacts to vCard
+Export contacts that are stored to the device internal memory to vCard
```
Comment (sledges) [inline Reference/Sailfish_OS_Cheat_Sheet/README.md:544]:
```suggestion
Export contacts that are stored in the device internal memory to vCard
```
IMHO `to ... to ...` makes it hard to parse, at least for me:)

**Review by rainemak — APPROVED** [review]

**Review by rainemak — APPROVED** [review]
@pherjung thank you for this clarification! :+1:

**Comment by sledges** [comment]
Please squash before merging.. next time:)

Pity that in GitHub Web UI "Squash and merge" doesn't create the merge commit though..
